src/Layout.tsx

```diff
@@ -1,5 +1,5 @@
 import { lazy, Suspense } from 'react'
-import Alert from 'src/components/common/Alert'
+import Alert from 'src/components/cms/common/Alert'
```

Comment:
This is weird. We have a CMS component affecting non CMS pages, since Layout.tsx is being imported on `gatsby-browser.tsx`.